src/include/daos/checksum.h

```diff
@@ -95,6 +95,16 @@ struct dcs_iod_csums {
 	uint32_t		 ic_nr;
 };
 
+/** Single value layout info for checksum */
+struct dcs_singv_layout {
```

Comment:
I understand this is only used by single value, but I think it's probably better to just call it dcs_layout, because we might have customized object class to support arbitrary EC stripe size in the future, so we might need this for EC object as well.